frail/verilog_printer.py

```diff
@@ -156,8 +156,10 @@ def print_verilog(e: AST, root: bool = True, lake_state: LakeDSLState = default_
             if port.input_dir:
                 io_strs[cur_scan_idx] += tab_str + f"input logic [{port.width - 1}:0] {port.name},\n"
             else:
-                io_strs[cur_scan_idx] = tab_str + "input logic clk, \n" + \
-                                        tab_str + f"output logic [{port.width - 1}:0] {port.name},\n" + io_strs[cur_scan_idx]
+                add_io_str = tab_str + "input logic clk, \n" + \
```

Comment:
So were we seeing a scan have multiple outputs?
Shouldn't memoization caught this case?